File: core/src/org/tetawex/ecf/model/LevelData.java
package org.tetawex.ecf.model;

/**
 * Created by Tetawex on 06.06.2017.
 */
public class LevelData {
    private Cell[][] cellArray;
    private int mana;
    private int maxScore;
    private int levelNumber;
    private String levelCode;
    private String name;

    public LevelData() {
    }

    public LevelData(Cell[][] cellArray, int mana, int maxScore, int levelNumber, String name, String levelCode) {
        this.cellArray = cellArray;
        this.mana = mana;
        this.maxScore = maxScore;
        this.levelNumber = levelNumber;
        this.name = name;
        this.levelCode=levelCode;
    }

    public Cell[][] getCellArray() {
        return cellArray;
    }

    public void setCellArray(Cell[][] cellArray) {
        this.cellArray = cellArray;
    }

    public int getMana() {
        return mana;
    }

    public void setMana(int mana) {
        this.mana = mana;
    }

    public int getMaxScore() {
        return maxScore;
    }

    public void setMaxScore(int maxScore) {
        this.maxScore = maxScore;
    }

    public int getLevelNumber() {
        return levelNumber;
    }

    public void setLevelNumber(int levelNumber) {
        this.levelNumber = levelNumber;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getLevelCode() {
        return levelCode;
    }

    public void setLevelCode(String levelCode) {
        this.levelCode = levelCode;
    }
}

File: core/src/org/tetawex/ecf/model/LevelData.kt
package org.tetawex.ecf.model

/**
 * Created by Tetawex on 06.06.2017.
 */
class LevelData {
    lateinit var cellArray: Array<Array<Cell?>>
    var mana: Int = 0
    var maxScore: Int = 0
    var levelNumber: Int = 0
    var levelCode: String? = null
    var name: String? = null

    constructor() {}

    constructor(cellArray: Array<Array<Cell?>>, mana: Int, maxScore: Int, levelNumber: Int, name: String, levelCode: String) {
        this.cellArray = cellArray
        this.mana = mana
        this.maxScore = maxScore
        this.levelNumber = levelNumber
        this.name = name
        this.levelCode = levelCode
    }
}
